core/runtime/src/main/java/io/quarkus/runtime/logging/LogMetricsHandler.java

```diff
@@ -0,0 +1,39 @@
+package io.quarkus.runtime.logging;
+
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.logging.Handler;
+import java.util.logging.LogRecord;
+
+/**
+ * Measures the number of log messages based on logger configurations quarkus.log.level and quarkus.log.category.*.level
+ * <p>
+ * It should reflect the values of the handler that logs the most, since best practice is to align its level with the root
+ * level.
+ */
+public class LogMetricsHandler extends Handler {
+
+    final Map<Integer, AtomicLong> logCounters;
```

Comment:
I was just having a peek, not an expert of this area but I'm wondering:
shouldn't such code prefer using a `LongAdder` over and `AtomicLong` ?

Reply:
Good point !